### ISSUE TITLE
Not working on Android

### ISSUE THREAD

**Issue by bastaware** [react-dnd/react-dnd#1660]
**Describe the bug**
Been testing on desktop and IPad - works like a charm.
Trying on Huawei MediaPad T5, Android 8.0.0. It's not possible to drag items, as the screen just starts scrolling when dragging.

Since this is not reported before, I assume that it's a device / android-version specific thing?

**Reproduction**

1. Open Chrome on Android (Huawei MediaPad)
2. Go to http://localhost:8000/examples/dustbin/single-target
3. Try to drag one of the items e.g. Paper
4. dustbin turns yellow, but item can't be dragged.

**Expected behavior**
4. dustbin turns yellow and item can be dragged around without screen scrolling

**Screenshots**

**Desktop (please complete the following information):**

Android 8.0.0
Chrome 78.0.3904.108

**Additional context**

**Comment by ngquangan** [comment]
Did you have any solutions? Can you share it with me?

**Comment by bit-scrambler** [comment]
I guess the demo is using HTML5Backend, instead it should be used TouchBackend as noted here:
https://react-dnd.github.io/react-dnd/docs/backends/touch

**Comment by bastaware** [comment]
Exactly. I switched to react-dnd-multi-backend and that fixed the problem. The thing that fooled me, is that IOS works with HTML5Backend.

Now I'm struggling with scrolling while dragging (also Android). But that's an entirely different matter ;)

**Comment by ngquangan** [comment]
Thanks all. I will try it

**Comment by getnot** [comment]
@bastaware did you find any solution for 
"Now I'm struggling with scrolling while dragging (also Android). But that's an entirely different matter ;)"

**Comment by bastaware** [comment]
> @bastaware did you find any solution for "Now I'm struggling with scrolling while dragging (also Android). But that's an entirely different matter ;)"

Sorry, I can't remember 🤔

**Comment by 55Cancri** [comment]
> @bastaware did you find any solution for "Now I'm struggling with scrolling while dragging (also Android). But that's an entirely different matter ;)"

Yes it seems REACT-DND DOES NOT WORK ON ANDROID EVEN WITH TOUCHBACKEND AS OF OCT 2023
